js/components/Tabs.js

```diff
@@ -29,13 +27,17 @@ function Tabs({ tabs, selectTab, loadSources, newSource }) {
   }
 
   return dom.ul(
-    {className: "tabs"},
-    tabsArr.map(tab => {
-      return dom.li({ "className": "tab",
-                      "data-actor-id": tab.actor,
-                      "onClick": onClickTab },
-        dom.div({ className: "tab-title" }, tab.title),
-        dom.div({ className: "tab-url" }, tab.url)
+    { className: "tabs" },
+    tabs.valueSeq().map(tab => {
+      return dom.li(
+        { "className": "tab",
+          "data-actor-id": tab.get("actor"),
+          "onClick": onClickTab },
+        dom.div(
+          { className: "tab-title" },
+          tab.get("title"),
+          dom.div({ className: "tab-url" }, tab.get("url"))
+        )
```

Comment:
I just realized I think I messed up the structure here. `tab-url` shouldn't be inside of `tab-title`. Doh! You're structure before was pretty good but the main thing was moving the props down. Otherwise my JS formatter wants to bump the rest of the args over to line up with it...

Anyway, I'll make a new commit that fixes the structure